ReactAndroid/src/main/java/com/facebook/react/modules/systeminfo/AndroidInfoHelpers.java

```diff
@@ -56,4 +62,9 @@ private static String getServerIpAddress(int port) {
 
     return String.format(Locale.US, "%s:%d", ipAddress, port);
   }
+
+  public static boolean isRunningOnTV(ReactContext reactContext) {
+    UiModeManager uiModeManager = (UiModeManager) reactContext.getSystemService(UI_MODE_SERVICE);
+    return uiModeManager.getCurrentModeType() == Configuration.UI_MODE_TYPE_TELEVISION;
```

Comment:
Add a comment with link to this:
https://developer.android.com/training/tv/start/hardware.html#runtime-check